sdktests/server_side_persistence_base.go

```diff
@@ -572,72 +572,10 @@ func (s *ServerSidePersistentTests) Run(t *ldtest.T) {
 
 					require.NoError(t, s.persistentStore.WriteMap(s.defaultPrefix, "features", s.initialFlags))
 
-					if cacheConfig.Mode == servicedef.CacheModeInfinite {
-						// This key was already cached, so it shouldn't see the change above.
-						h.RequireNever(t,
-							checkForUpdatedValue(t, client, "flag-key", context,
-								ldvalue.String("value"), ldvalue.String("new-value"), ldvalue.String("default")),
-							time.Millisecond*500, time.Millisecond*20, "flag-key was incorrectly updated")
-
-						// But since we didn't evaluate this flag, this should actually be
-						// reflected by directly changing the database.
-						h.RequireEventually(t,
-							checkForUpdatedValue(t, client, "uncached-flag-key", context,
-								ldvalue.String("default"), ldvalue.String("fallthrough"), ldvalue.String("default")),
-							time.Millisecond*500, time.Millisecond*20, "uncached-flag-key was incorrectly cached")
-					} else if cacheConfig.Mode == servicedef.CacheModeTTL {
-						// This key was already cached, so it shouldn't see the change above.
-						h.RequireNever(t,
-							checkForUpdatedValue(t, client, "flag-key", context,
-								ldvalue.String("value"), ldvalue.String("new-value"), ldvalue.String("default")),
-							time.Duration(
-								int(time.Second)*cacheConfig.TTL.Value()/2),
-							time.Millisecond*20,
-							"flag-key was incorrectly updated")
-
-						// But eventually, it will expire and then we will fetch it from the database.
-						h.RequireEventually(t,
-							checkForUpdatedValue(t, client, "flag-key", context,
-								ldvalue.String("value"), ldvalue.String("fallthrough"), ldvalue.String("default")),
-							time.Duration(int(time.Second)*cacheConfig.TTL.Value()), time.Millisecond*20, "flag-key was incorrectly cached")
-					}
-				})
-
-				s.runWithEmptyStore(t, "ignores dropped flags", func(t *ldtest.T) {
```

Comment:
This is redundant with the check above. Whether I delete flags or change them, it is confirming the same thing.